### PR TITLE
Update v5.2.2

### PR DESCRIPTION
**Pull request by micxer**

### CODE REVIEW

File: templates/elasticsearch.yml.j2 [hunk 1]
```diff
@@ -93,5 +93,3 @@ http.port: {{ elasticsearch_http_port }}
 # Require explicit names when deleting indices:
 #
 # action.destructive_requires_name: true
-script.inline: {{ elasticsearch_script_inline }}
-script.indexed: {{ elasticsearch_script_indexed }}
```
Comment (micxer) [inline templates/elasticsearch.yml.j2]: Why are these removed?

Comment (tkramheller) [inline templates/elasticsearch.yml.j2]: old config for elasticsearch 2.3
